core/object/script_language.cpp

```diff
@@ -34,6 +34,7 @@
 #include "core/core_string_names.h"
 #include "core/debugger/engine_debugger.h"
 #include "core/debugger/script_debugger.h"
+#include "core/object/object.h"
```

Comment:
```suggestion
```
Not needed, `PlaceholderScriptInstance` derives from `Object` and it is included already (if you are using a "include what you use" feature in your editor please turn it off when working with Godot)

Reply:
Thanks for the tip 🙏  
I switched to clangd recently and didn't notice that was the default behaviour!